extension/src/experiments/model/filterBy/quickPick.ts

```diff
@@ -21,6 +21,12 @@ export const OPERATORS = [
     types: ['number', 'string'],
     value: Operator.NOT_EQUAL
   },
+  {
+    description: 'Not Missing',
```

Comment:
Would an antonym of missing make more sense?  Like "Defined" or "Contains"?

Reply:
"Missing"/"Missing values" is a Data Science term that most users will be familiar with.